P/PulseAudio/build_tarballs.jl

```diff
@@ -0,0 +1,79 @@
+cd $WORKSPACE/srcdir
```

Comment:
I think this file has some problems as it is now

Reply:
Oops. I added the full file. Still doesn't work on master due to #1001 but otherwise I think this should be good to go.

Reply:
Though bluetooth support might be nice to have. I've having a bit of trouble with bluez: even with eudev, I'm having trouble building ical? I'm up to

```
https://github.com/libical/libical/releases/download/v3.0.9/libical-3.0.9.tar.gz

# dependencies
Glib_jll
XML2_jll
GTK3_jll
ICU_jll
Perl_jll

# script
apk add gtk-doc
apk add doxygen
apk add libical=3.0.9-r0
cd libical-*
cmake -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release -DICAL_GLIB=False -DWITH_CXX_BINDINGS=False
make
make install
```